pkg/util/env/env.go

```diff
@@ -24,7 +24,10 @@ import (
 )
 
 const (
-	NodeNameEnvKey        = "NODE_NAME"
+	NodeNameEnvKey = "NODE_NAME"
+	// ExternalNodeEnvKey is an environment variable used in ExternalNode feature to tell the expected name with which
+	// antrea agent is running. This is also used in the shell where antctl is running on a VM host.
+	ExternalNodeEnvKey    = "EXTERNAL_NODE"
```

Comment:
Is it just a name string? If so, why not reuse "NODE_NAME"?

Reply:
This is mainly for antctl support. I need a different env variable to tell antctl it is running on a VM but not a K8s Node.

Reply:
Do we assume antctl always runs in a container? If not, the env will not be set anyway right?

I feel better to use another env variable to indicate node type for antctl, and also add a parameter for antctl for the case it runs without an env.